app/scripts/views/settings.js

```diff
@@ -150,9 +150,7 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
     },
 
     beforeDestroy: function () {
-      $('body.settings').fadeOut(this.FADE_OUT_SETTINGS_MS, function (){
-        $('body').removeClass('settings').show();
-      });
+      $('body').removeClass('settings');
```

Comment:
Ah, we need to add `body.settings #main-content.card` as another selector for this CSS rule:
https://github.com/mozilla/fxa-content-server/blob/master/app/styles/modules/_settings.scss#L21